goosebit/db/db.py

```diff
@@ -0,0 +1,15 @@
+from tortoise import Tortoise, run_async
```

Comment:
I think this should be put in `__init__.py`?  Thoughts?

Reply:
Not what I meant, I mean like I would prefer to just `from goosebit.db import init` rather than `from goosebit.db.db import init` or `from goosebit.db import db`, that seems strange to have a file structure like that.

Basically: rename `db.py` to `__init__.py`.

Reply:
Regarding the commit also, that `init()` call wasn't entirely redundant, it was so you could run that file manually and have it attempt to init the DB (mostly for testing/inital startup).  Probably fine to remove it now though.

Reply:
Renamed as proposed.